pandas/tests/series/test_io.py

```diff
@@ -82,6 +82,14 @@ def test_to_csv_unicode_index(self):
 
         assert_series_equal(s, s2)
 
+    def test_tolist_np(self):
+        # GH10904
+        np_array = np.array([1, 2, 3])
```

Comment:
pls check for all dtypes (well the numeric ones). Further construct against an absolute test, rather than again what numpy does. E.g. you want to check that things are actually python integers, floats etc.